.github/workflows/npm-update.yml

```diff
@@ -21,5 +21,5 @@ jobs:
 
       - name: Run npm-update bot
         run: |
-          make bots
+          tools/make-bots
```

Comment:
Ouch, thanks! This worked fine in starter-kit with its static Makefile, but I seem to have missed it :crying_cat_face:

Reply:
Ah, yes please, less confusing.